### PR TITLE
Support the StoRM-Tape REST API endpoint

### PR DESCRIPTION
**Pull request by giacomini**
StoRM-Tape doesn't need to be notified about recall failures, since it deduces recall success/failure based on stubbification and/or the presence of the TSMRecT xattr. For this reason it returns a taskid named "unused" for compatibility with the current protocol; however this taskid is not added to TSMRecT.

Note that StoRM-Tape creates an empty TSMRecT (if it doesn't exist yet) before passing a file to GEMSS.

### CODE REVIEW

**Comment by efattibene** [comment]
Some notes related to the request:
1. It would be better to add a new variable (i.e. STORM_TAPE_TOKEN) to file hsmConfig, to distinguish between authentication on STORM_TAPE_NODE and the one on STORM_BACKEND_NODE.
2. The error message at row 76 of yamssReorderrecall should be somethig like "Error: StoRM tape service not responding".
3. Il would be better to a add a new variable (i.e. RECALL_USE_STORM_TAPE) to file hsmConfig, since it could be possible to use only one of the two possible protocols at a time. In this case the code should be adapted accordingly.
4. The file tasks.txt should be removed at the end of the process.

File: scripts/yamssReorderRecall [hunk 1]
```diff
@@ -145,11 +157,14 @@ if [ "$RECALL_USE_STORM"x = "1x" -a "$numtakeover"x != "0x" ]; then
         continue
      fi
 
-     # if the file was already under recall, append taskid to the existing extended attribute
-     if [ "$cattr"x != x ]; then
-        nattr=$cattr","$taskid
-     else
-        nattr=$taskid
+     # the storm tape rest api doesn't need to be notified about failures
+     if [ "$taskid"x != "unused" ]; then
```
Comment (federicaagostini) [inline scripts/yamssReorderRecall]: "unused" -> "unused"x

**Comment by efattibene** [comment]
Ok. I would just add "":
STORM_TAPE_USER="someuser"
STORM_TAPE_PASSWORD="somepassword"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 gemss
 =====
+Tool to interface Storm with TSM.
```
Comment (giacomini) [inline README.md]: StoRM!

**Review by afalabel — APPROVED** [review]

**Pull request merged**